lib/classification_lifecycle.rb

```diff
@@ -23,7 +23,7 @@ def initialize(classification, action)
   end
 
   def execute
-    return if create? && classification.lifecycled_at.present?
+    return if action == "create" && classification.lifecycled_at.present?
```

Comment:
do we even need the action passed in now? Should we just rely on the state inspection instead of short circuiting?